### ISSUE TITLE
Make HEAP and JMX OPTS configurable

### ISSUE THREAD

**Issue by samklr** [confluentinc/cp-ansible#34]
Hello,
This project is awesome for people getting started to roll out a Kafka cluster into production.
One issue though is trying to set KAFKA_HEAP_OPTS and JMX_OPTS into broker configurations.
Tried to add 
environment:
      KAFKA_HEAP_OPTS: '-Xms4g -Xmx4g .... '
      JMX_PORT: 9999
      KAFKA_JMX_OPTS: '-.. '
into the Broker configuration role, but it doesn't seem to be taken into account. 
So ended up do it manually. Any pointer on this.

Would also be nice to have a restart and uninstall role.

Sorry for rambling, thanks again for your help.

**Comment by cotedm** [comment]
Looks like we need to add a way to pass the environment variables into the systemd services. This issue will be used to make that happen. The restart and uninstall roles I'll open new issues for, makes sense to me!

**Comment by cricket007** [comment]
@cotedm 

There was [KAKFA-1566](https://issues.apache.org/jira/browse/KAFKA-1566), not sure what happened to it, but it would enable putting `EnvironmentFile` as a systemd directive. 

Otherwise, you would need to template out the systemd scripts to add something like

```
{% for k,v in service.environment.items() %}
Environment={{k}}={{v}}
{% endfor %}
```

**Comment by cricket007** [comment]
The other option would just template the `EnvironmentFile` in this project, then use [`lineinfile`](https://docs.ansible.com/ansible/2.5/modules/lineinfile_module.html), and replacing `[Service]` with `[Service]\nEnvironmentFile={{service}}-env.sh`

**Comment by cotedm** [comment]
@cricket007 yes that issue appears abandoned, so probably not a good option near term. I'm exploring if we can do a drop in and continue using the Confluent provided systemd scripts or use the `systemctl edit` style with ansible in some way. It seems the ansible systemd module doesn't support the environment or environmentfile directive. If that doesn't work out, I think templating the environment file and then referencing it from the on systemd scripts on the fly is the only way to go. A bit uglier.
